server/chat-commands/admin.ts

```diff
@@ -313,19 +313,24 @@ export const commands: Chat.ChatCommands = {
 		`/pmuhtmlchange [user], [name], [html] - Changes html that was previously PMed to [user] to [html]. Requires * # &`,
 	],
 
+	closehtmlpage: 'sendhtmlpage',
 	changehtmlpageselector: 'sendhtmlpage',
 	sendhtmlpage(target, room, user, connection, cmd) {
 		room = this.requireRoom();
 		this.checkCan('addhtml', null, room);
 
+		const closeHtmlPage = cmd === 'closehtmlpage';
+
 		const {targetUser, rest} = this.requireUser(target);
 		let [pageid, content] = this.splitOne(rest);
 		let selector: string | undefined;
 		if (cmd === 'changehtmlpageselector') {
 			[selector, content] = this.splitOne(content);
 			if (!selector) return this.parse(`/help ${cmd}`);
 		}
-		if (!pageid || !content) return this.parse(`/help ${cmd}`);
+		if (!pageid || (closeHtmlPage ? content : !content)) {
```

Comment:
This can actually be further simplified to `closeHtmlPage === !!content`, but I think the `?:` method is most readable.

Reply:
This is the one that lets me read "if it's `/closehtmlpage`, having content is an error, otherwise, not having content is the error".